index.js

```diff
@@ -78,6 +83,196 @@ function loader (version) {
         return new Item(item.blockId, item.itemCount, item.itemDamage, item.nbtData)
       }
     }
+
+    static getEnchants (item) {
+      const normalize = (enchList) => enchList.map(ench => {
+        return { lvl: ench.lvl, name: mcData.enchantments[ench.id].name }
+      })
+      if (mcData.isOlderThan('1.13')) {
+        let itemEnch
+        if (item.name === 'enchanted_book' && item.nbt !== null) {
+          itemEnch = nbt.simplify(item.nbt).StoredEnchantments
+        } else if (item.nbt !== null) {
+          itemEnch = nbt.simplify(item.nbt).ench
+        } else {
+          itemEnch = []
+        }
+        return normalize(itemEnch)
+      }
+      const gg = nbt.simplify(item.nbt)
+      console.log(gg)
+      return 1
```

Comment:
So if the data is newer than 1.13, you don't return anything ?

Reply:
still a wip

Reply:
done